packages/cdktf-cli/src/bin/cmds/handlers.ts

```diff
@@ -236,38 +237,47 @@ export async function get(argv: {
   language: Language;
   parallelism: number;
   force?: boolean;
+  showPerformanceInfo?: boolean;
 }) {
-  throwIfNotProjectDirectory();
-  await displayVersionMessage();
-  await initializErrorReporting(askForCrashReportingConsent);
-  await checkEnvironment();
-  await verifySimilarLibraryVersion();
-  const config = readConfigSync(); // read config again to be up-to-date (if called via 'add' command)
-  const providers = config.terraformProviders ?? [];
-  const modules = config.terraformModules ?? [];
-  const { output, language, parallelism, force } = argv;
-
-  const constraints: TerraformDependencyConstraint[] = [
-    ...providers.map((c) => new TerraformProviderConstraint(c)),
-    ...modules.map((c) => new TerraformModuleConstraint(c)),
-  ];
-
-  if (constraints.length === 0) {
-    logger.warn(
-      `WARNING: No providers or modules found in "cdktf.json" config file, therefore cdktf get does nothing.`
+  const printPerformanceInfo = argv.showPerformanceInfo
```

Comment:
This `printPerformanceInfo = startPerformanceMonitoring()` makes me feel odd, but I also don't have a much nicer solution here.